float32[2] pos # x, y and orientation m m deg
float32[2] vel #vx vy
float32[] cluster_pos # x1 y1 x2 y2 and so on
float32[2] cluster_vel # x1 y1 x2 y2 and so on
float32[] obstacle # x1 y1 r1 x2 y2 r2
float32 orientation # deg
bool stop # when we reach waypoint stop the robot movement
